src/iop/toneequal.c

```diff
@@ -1825,6 +1825,10 @@ static void switch_cursors(struct dt_iop_module_t *self)
     // if pipe is clean and idle and cursor is on preview,
     // hide GTK cursor because we display our custom one
     dt_control_change_cursor(GDK_BLANK_CURSOR);
+    dt_control_hinter_message(darktable.control,
+                              "scroll over image to change tone exposure by 0.25 EV\n"
```

Comment:
The numbers should not be there as this could change anytime.

Reply:
I would rewrite this to something like that:

```
scroll over image to change tone exposure
use shift+scroll  for larger steps
use ctrl+scroll for smaller steps.
```

Reply:
this is done.